test/e2e/core.go

```diff
@@ -82,6 +82,7 @@ func CoreDump(dir string) {
 		cmds = append(cmds, []command{
 			{"cat /var/log/kubelet.log", "kubelet"},
 			{"cat /var/log/supervisor/supervisord.log", "supervisord"},
+			{"cat /var/log/dmesg", "dmesg"},
```

Comment:
/var/log/dmesg is only written right after boot, I think. Maybe /var/log/messages?